shared/src/store/action-reducers/utils/calculate-treatments.spec.ts

```diff
@@ -263,34 +270,30 @@ describe('calculate treatment', () => {
         ]);
     });
 
-    it('treats the patient with worse status within the generalThreshold, regardless of distance', () => {
+    it('treats the patient with worse status within the treatmentRange, regardless of distance', () => {
         const ids = {
             material: '',
             greenPatient: '',
             redPatient: '',
         };
         const { beforeState, newState } = setupStateAndApplyTreatments(
             (state) => {
-                const greenPatient = generatePatient(
+                ids.greenPatient = addPatient(
+                    state,
                     'green',
                     'green',
-                    Position.create(-1, -1)
-                );
-                const redPatient = generatePatient(
+                    Position.create(-3, -3)
+                ).id;
+                ids.redPatient = addPatient(
+                    state,
                     'red',
                     'red',
-                    Position.create(2, 2)
-                );
-                const material = generateMaterial(Position.create(0, 0));
-
-                ids.material = material.id;
-                ids.greenPatient = greenPatient.id;
-                ids.redPatient = redPatient.id;
-                state.patients[greenPatient.id] = greenPatient;
-                state.patients[redPatient.id] = redPatient;
-                state.materials[material.id] = material;
+                    Position.create(3, 3)
+                ).id;
+                ids.material = addMaterial(state, Position.create(0, 0)).id;
             }
         );
+        console.log(ids);
```

Comment:
Just saw this log in a test run. Is it required for anything, or could we remove this sometime?

Reply:
I think this can be removed. This PR was so big that we completely missed that.

Reply:
:thumbsup: This should probably not be in there.